src/screens/Apps/Lendasat/Index.tsx

```diff
@@ -0,0 +1,202 @@
+import { useContext, useEffect, useRef, useState } from 'react'
+import Padded from '../../../components/Padded'
+import Header from '../../../components/Header'
+import Content from '../../../components/Content'
+import FlexCol from '../../../components/FlexCol'
+import { SettingsIconLight } from '../../../icons/Settings'
+import { NavigationContext, Pages } from '../../../providers/navigation'
+import { WalletContext } from '../../../providers/wallet'
+import { AddressType, type LoanAsset, WalletProvider } from '@lendasat/lendasat-wallet-bridge'
+import { sha256 } from '@noble/hashes/sha2.js'
+import * as utils from '@noble/hashes/utils.js'
+import * as secp from '@noble/secp256k1'
+import { secp256k1 } from '@noble/curves/secp256k1.js'
+import { hmac } from '@noble/hashes/hmac.js'
+import { collaborativeExit, getReceivingAddresses } from '../../../lib/asp'
+import { Transaction } from '@scure/btc-signer'
```

Comment:
```suggestion
import { Transaction } from '@arkade-os/sdk'
```
We re-export Transaction class with all default required by the protocol